examples/online-publication-with-product-lines.json

```diff
@@ -0,0 +1,1376 @@
+{
```

Comment:
Why is a separate file needed? Can't we add `teaser` to [examples/online-publications.json](https://github.com/detailnet/denner-portal-api-spec/blob/master/examples/online-publications.json)?

Reply:
It makes sense.

The reason to make the new one, is because I've taken a real online-publication .. to provide a correct example

I'll integrate all in the same

Reply:
@curzio-della-santa, it would be good to have a PR with only the `teaser` as diff, so that we can send the link to Visual Fantastix (they might get confused when there's a whole lot of unrelated changes).